public/client.js

```diff
@@ -0,0 +1,165 @@
+/* global SimplePeer */
+const MAX_RECONNECT_ATTEMPTS = 30;
+
+const WEBSOCKET_MESSAGE_HANDLERS = {
+  'signal-data': handleSignalData,
+  'new-player': handleNewPlayer,
+  'room-joined': handleRoomJoined,
+  'room-join-error': handleRoomJoinError,
+  'client-id': handleClientId
+};
+
+function handleClientId (clientId) {
+  window.PacMen.clientId = clientId;
```

Comment:
This `window` usage is fine for now, I want us using the same ES6 code in the browser soon.